app/controllers/draws_controller.rb

```diff
@@ -91,38 +91,26 @@ def students_update
     handle_action(**result)
   end
 
-  def process_bulk_assignment
-    result = DrawStudentsUpdate.update(draw: @draw,
-                                       params: students_update_params)
-    @students_update = result[:update_object]
-    if @students_update
-      prepare_students_edit_data
-      result[:action] = 'student_summary'
-    else
-      result[:path] = student_summary_draw_path(@draw)
+  def start_lottery
+    @lottery_starter = DrawLotteryStarter.new(draw: @draw)
+    if handle_oversubscription?
```

Comment:
can we actually just conditionally change the "Proceed to Lottery" button to a "Proceed to Oversubscription" instead of this?